keras_cv/layers/preprocessing/random_crop_and_resize.py

```diff
@@ -145,6 +148,62 @@ def augment_image(self, image, transformation, **kwargs):
     def augment_target(self, target, **kwargs):
         return target
 
+    def _transform_bounding_boxes(bounding_boxes, transformation):
+        t_y1, t_x1, t_y2, t_x2 = transformation[0]
+        t_dx = t_x2 - t_x1
+        t_dy = t_y2 - t_y1
+        x1, y1, x2, y2, rest = tf.split(
+            bounding_boxes, [1, 1, 1, 1, bounding_boxes.shape[-1] - 4], axis=-1
+        )
+        output = tf.stack(
+            [
+                (x1 - t_x1) / t_dx,
+                (y1 - t_y1) / t_dy,
+                (x2 - t_x1) / t_dx,
+                (y2 - t_y1) / t_dy,
+                rest,
+            ],
+            axis=-1,
+        )
+        output = tf.squeeze(output, axis=1)
```

Comment:
I think that you should be able to get rid of this squeeze by stacking on the correct axis.  If you need help debugging feel free to tf.print the shapes out and add them in code comments, I can try to help you structure it to fix this.

Reply:
Solved in a new commit (abb53b0d02a586f4eaf5be540f35c329cd0ff955)